generator/integration_tests/BUILD.bazel

```diff
@@ -59,6 +59,70 @@ cc_grpc_library(
     name = "google_cloud_cpp_generator_testing_cc_grpc",
     srcs = [":google_cloud_cpp_generator_testing_proto"],
     grpc_only = True,
-    visibility = [":__subpackages__"],
     deps = [":google_cloud_cpp_generator_testing_cc_proto"],
 )
+
+filegroup(
+    name = "golden_srcs",
+    srcs = glob([
+        "golden/*.cc",
+        "golden/internal/*.cc",
+    ]),
+)
+
+filegroup(
+    name = "golden_hdrs",
+    srcs = glob([
+        "golden/*.h",
+        "golden/internal/*.h",
+        "golden/mocks/*.h",
```

Comment:
should these be split out into their own `google_cloud_cpp_generator_golden_mocks` library?

Reply:
I will leave these as-is until I find a reason to split them.